packages/plugin-ext/src/plugin/plugin-context.ts

```diff
@@ -1124,7 +1130,10 @@ export function createAPIFactory(
             TestTag,
             TestRunRequest,
             TestMessage,
-            ExtensionKind
+            ExtensionKind,
+            InlineCompletionItem: InlineCompletionItem,
+            InlineCompletionList: InlineCompletionList,
```

Comment:
Minor:
```suggestion
            InlineCompletionItem,
            InlineCompletionList,
```

Reply:
Thanks, the result of just doing a rename :)